addons/account/models/mail_scheduled_message.py

```diff
@@ -0,0 +1,176 @@
+import json
+
+from odoo import api, fields, models, SUPERUSER_ID, _
+from odoo.fields import Datetime
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class MailScheduledMessage(models.Model):
+    _inherit = 'mail.scheduled.message'
+
+    from_move_send_wizard = fields.Boolean(compute='_compute_from_move_send_wizard')
```

Comment:
Small proposal (wizard suffix not necessary, let us use the model name instead so that it is easy to find)
```suggestion
    from_account_move_send = fields.Boolean(compute='_compute_from_move_send_wizard')
```